beps/docs/proposals/BEP-009-function-and-lambda-syntax-design-ts-aligned/README.md

```
@@ -0,0 +1,678 @@
+  code => "Unknown: " + code.toString()
+}
+```
+
+The `=>` token is used for both lambda expressions and match arms. Context always disambiguates: within a `match` block, `=>` separates a pattern from its result expression; outside a `match` block, `=>` introduces a lambda body.
+
+When a match arm produces a lambda, both uses appear on the same line:
+
+```ts
+match mode {
+  "add" => (x: int) => x + 1
+  "sub" => (x: int) => x - 1
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)